tests/baselines/reference/grammarErrors.js

```diff
@@ -12,7 +12,24 @@ import type A, { B, C } from './a';
 import type A from './a';
 export type { A };
 
+//// [c.ts]
+namespace ns {
+  export class Foo {}
+}
+import type Foo = ns.Foo;
+
 
 //// [b.js]
 "use strict";
 exports.__esModule = true;
+//// [c.js]
+var ns;
+(function (ns) {
+    var Foo = /** @class */ (function () {
+        function Foo() {
+        }
+        return Foo;
+    }());
+    ns.Foo = Foo;
+})(ns || (ns = {}));
+var Foo = ns.Foo;

```

Comment:
Although this line comes from a grammar error, common sense dictates that it shouldn’t emit anyway. I need to double check all the emit before merging, because I don’t recall doing anything extra here to make sure valid `import type` requires get elided.